qiskit/extensions/standard/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2017.
+# (C) Copyright IBM 2020.
```

Comment:
I don't think this is correct to do. The copyright date of 2017 is when the module was written adding one line doesn't change that, You can probably add 2020 here or do a range.

Reply:
Yes, you're right. I was always confused about how this is correctly done. Do you know whether either of `2017-2020` or `2017, 2020` is more correct? Currently, most codes I've seen in Qiskit use the comma-separation.

Reply:
@nonhermitian can confirm policy, but I believe the when editing files you leave the original date.
and new files you use `(C) Copyright IBM 2017, YEAR FILE ADDED`

Reply:
Okay, thanks for the clarification. I reverted the years to what had been in place previously. 👍